slither/core/expressions/identifier.py

```diff
@@ -78,3 +78,6 @@ def value(
 
     def __str__(self) -> str:
         return str(self._value)
+
+    def expression(self):
+        return self
```

Comment:
Should this be decorated as `@property`?
```suggestion

    @property
    def expression(self):
        return self
```

Reply:
Good catch, I missed that!